docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,242 @@
+# Sentinel-1 InSAR Product Guide
+
+## Introduction
+Interferometric SAR (InSAR) processing uses two SAR images of the same area to determine geometric properties of the surface. With InSAR, Digital elevation models (DEMs) can be routinely created (SRTM, GLO-30). Moreover, two or more pairs of images can be used to extract surface motion or deformation at the millimeter level scale.
```

Comment:
I'd suggest that we focus on the surface deformation applications of InSAR. I would be hesitant to even mention DEM generation in a Sentinel-1 product guide, especially in the intro. This type of mission is not well-suited for DEM generation, and while it might be worth mentioning somewhere deep down in the document, I would suggest removing the mention here.
Franz would prefer that we not even have the data recipe available for generating DEMs from S1 data.

Reply:
I removed the mention of DEM creation